editor/effects.js

```diff
@@ -388,4 +390,8 @@ export default {
 		dispatch( saveReusableBlock( reusableBlock.id ) );
 		dispatch( replaceBlocks( [ oldBlock.uid ], [ newBlock ] ) );
 	},
+	APPEND_DEFAULT_BLOCK( action, store ) {
+		const { dispatch } = store;
+		dispatch( insertBlock( createBlock( getDefaultBlockName() ) ) );
```

Comment:
Minor: Effects can simply `return` an action object to dispatch, if they're synchronous.

Reply:
Ah, good tip, easy to forget.